starlite/handlers/http.py

```diff
@@ -424,10 +424,10 @@ def resolve_response_headers(self) -> "ResponseHeadersMap":
         resolved_response_headers = {}
         for layer in self.ownership_layers:
             resolved_response_headers.update(layer.response_headers or {})
-        if self.cache_control:
-            resolved_response_headers.update(
-                {CacheControlHeader.CACHE_CONTROL_HEADER: ResponseHeader(value=self.cache_control.to_header())}
-            )
+            if cache_control := getattr(layer, "cache_control", None):
```

Comment:
Probably should be 

```python
cache_control = getattr(layer, "cache_control", None)
if cache_control:
   ...
```

To support Python < 3.8

Reply:
Thanks, fixed!